common/beerocks/bwl/CMakeLists.txt

```
@@ -17,7 +17,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
 
 # UGW
 if (TARGET_PLATFORM STREQUAL "ugw")
-    set(BWL_TYPE "DWPAL")
+   set(BWL_TYPE "DWPAL")
```

Incorrect whitespace change.

---

Just to be clear: my suggestion from below is to change this like this:
```suggestion
    set(BWL_TYPE_DEFAULT "DWPAL")
```